code/modules/mob/transform_procs.dm

```diff
@@ -78,9 +78,6 @@
 		O.setBrainLoss(getBrainLoss(), 0)
 		O.updatehealth()
 		O.radiation = radiation
-		for(var/T in get_traumas())

```

Comment:
Why these removals ?

Reply:
i forgot to remove these on the trauma PR, since after i ported everything to the brain it should carry over even when monkeying/unmonkeying. These would likely duplicate traumas every time someone monkeys/unmonkeys, i'm suprised nothing has come up yet.